debian/control.in

```diff
@@ -12,9 +14,19 @@ Homepage: http://www.openucx.org
 #Depends: ucx (= ${binary:Version})
 #Description: Header files for UCX library
 
-Package: @PACKAGE@
+Package: ucx
 Section: libs
-Depends: libc6, libgomp1, libnuma1
+Depends: ${misc:Depends}, ${shlibs:Depends}
 Architecture: any
 Description: Unified Communication X
  UCX is a communication library implementing high-performance messaging.
+
+Package: ucx-cuda
+Section: libs
+Depends: ${misc:Depends}, ${shlibs:Depends}
+Architecture: any
+Build-Profiles: <cuda>
+Description: Unified Communication X
+ UCX is a communication library implementing high-performance messaging.
+ .
+ This package includes CUDA support for UCX.
```

Comment:
Provide CUDA (NVIDIA GPU) support for UCX. Enables passing GPU memory pointers
to UCX communication routines, and transports taking advantage of GPU-Direct
technology for direct data transfer between GPU and RDMA devices.